### PR TITLE
Add support for Source objects

### PR DESCRIPTION
**Pull request by spolu**
Add support for Sources to Go.

### CODE REVIEW

**Comment by tejasmanohar** [comment]
+100

**Comment by spolu** [comment]
r? @brandur 
cc @andrewpthorp

**Comment by brandur** [comment]
Wow, crazy PR here! Thanks :)

LGTM.

**Comment by brandur** [comment]
Released as 17.3.0.

**Comment by spolu** [comment]
Thanks @brandur!!

File: address.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package stripe
+
+// Standard address parameters.
+type AddressParams struct {
+	Line1      string
+	Line2      string
+	City       string
+	State      string
+	PostalCode string
```
Comment (remistr) [inline address.go]: Why use `PostalCode` and not `Zip` like we do on Card?

Comment (spolu) [inline address.go]: ZIP is a US concept

**Pull request merged**